html-skeleton/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css">
+    <title>sample code</title>
+  </head>
+  <body>
+    <script src="main.js"></script>
```

Comment:
You seem to be missing the `Hello HTML!` message, and there is a `script` element here for no clear reason, as well as a `link` element in the `head` element. Please remove these lines and add the message requested in the exercise